tests/integration/suite/test_node.py

```diff
@@ -139,9 +139,9 @@ def test_writing_config_to_disk(admin_mc, wait_remove_resource):
         digitaloceancredentialConfig={"accessToken": "test"})
     wait_remove_resource(cloud_credential)
 
-    data = {'userdata': 'do cool stuff\n',
+    data = {'userdata': 'do cool stuff' + random_str() + '\n',
```

Comment:
Is `userdata` always expected to be just one line or does multiline input need to be tested here?

If multi-line input should be tested, should there be a `/n` at the end of `do cool stuff` (or whatever it is changed to)?

Reply:
`userdata` was already one line. I added the `random_str()` in the middle be testing on my local machine would always pass this test because the contents of the file was not changing.